root/about.html

```diff
@@ -18,12 +18,12 @@
 
 MetaCPAN has two parts:
 
-* [metacpan.org](https://metacpan.org), a front end to...
+* [v1.metacpan.org](https://v1.metacpan.org), a front end to...
 * [api-v1.metacpan.org](http://api-v1.metacpan.org/) the API
 
 So
-[https://metacpan.org/module/Moose](https://metacpan.org/module/Moose)
-vs [http://api-v1.metacpan.org/module/Moose](http://api-v1.metacpan.org/module/Moose)
+[https://v1.metacpan.org/module/Moose](https://v1.metacpan.org/module/Moose)
+vs [http://api.metacpan.org/module/Moose](http://api-v1.metacpan.org/module/Moose)
```

Comment:
Change the link http://api.metacpan.org/module/Moose to http://api-v1.metacpan.org/module/Moose